travis.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+echo "PW = $CGALTestPyPiPassword"
+set -e
+set -x
+export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
+pycodestyle --show-source --show-pep8 $PYCODESTYLE_CONVENTIONS $CGAL_PYTHON_EXAMPLES || pycodestyle --statistics $PYCODESTYLE_CONVENTIONS -qq $CGAL_PYTHON_EXAMPLES
+if [ $(python -c "import sys; print(sys.version_info.major)") = "3" ]  && [ "$TRAVIS_OS_NAME" = "linux" ]; then
+  python -m pip install --verbose wheelhouse/*
+  cd examples/python
+  for test_file in ./*.py; do python $test_file; done
+  cd ../..
+  twine upload -r testpypi wheelhouse/*  -u "maxgimeno" -p "$CGALTestPyPiPassword"
```

Comment:
I think `"$CGALTestPyPiPassword"` is empty, because the variable `CGALTestPyPiPassword` was set but not exported by the parent shell.